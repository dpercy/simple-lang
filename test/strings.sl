(ord " ")
(ord "A")
(chr 32)
(chr 65)
(ord "AB")
(string-length "AB")
(chr 0)
(ord "\0")
(chr 128)
