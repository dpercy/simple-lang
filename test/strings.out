32
65
" "
"A"
(error "ord: expected a character")
2
"\0"
0
(error "chr: expected an ASCII integer")
"\u{1}"
"\u{6}"